pkg/cmd/extension/manager_other_test.go

```diff
@@ -0,0 +1,36 @@
+//go:build !windows
```

Comment:
From @andyfeller @BagToad conversation:

- Need to merge in changes from https://github.com/cli/cli/pull/9933
- Make sure you are cleaning up the extension update directory

Reply:
In pulling these changes into the branch, I made a minor change for consistency with other PR changes. 👍